README.md

```diff
@@ -341,6 +354,25 @@ DateTimeConverter:
 
 ```
 
+## Inline types (since 6.0.0)
```

Comment:
I would put this above the default config

Reply:
The problem is, that we have so much above the default config, that you need to scroll down all the way just to see the default syntax. @vanlooverenkoen wdyt?

Reply:
Alright, maybe we should add a ticket to simplify the readme. The whole documentation shouldn't be in the readme, but be separate tbh

Reply:
https://github.com/icapps/flutter-model-generator/issues/115